src/stuf/stuf_zds/templates/stuf_zds/soap/creeerZaak.xml

```diff
@@ -100,6 +99,17 @@
                                 {% include "stuf_zds/soap/includes/bezoekadres.xml" %}
                             {% endif %}
                         </ZKN:nietNatuurlijkPersoon>
+                    {% elif initiator.handelsnaam %}
```

Comment:
This was also not as discussed. We discussed to make it similar to `ZKN:natuurlijkPersoon`. This is mostly usefull for simply inheriting all attributes for both auth and non-auth scenarios.

For example: the `verblijfsadres` is now missing in the non-auth scenario.